lib/banking_calendar/data/bsp.yml

```diff
@@ -109,5 +109,26 @@ bank_holidays:
   - 2024-12-08
   - 2024-12-24
   - 2024-12-25
+  - 2024-12-26
   - 2024-12-30
   - 2024-12-31
+  - 2025-01-01
+  - 2025-01-02
+  - 2025-01-29
+  - 2025-04-09
+  - 2024-04-17
+  - 2024-04-18
+  - 2024-04-19
+  - 2025-05-01
+  - 2025-06-12
+  - 2025-08-21
+  - 2025-08-25
+  - 2025-10-31
+  - 2024-11-01
+  - 2024-11-08
```

Comment:
Please double check this too.

Reply:
<img width="632" alt="Screenshot 2024-12-11 at 4 58 43 PM" src="https://github.com/user-attachments/assets/0db6f0ca-34a4-4890-a42b-4de4acc2e327">